src/main/java/io/xpring/common/XRPLNetwork.java

```diff
@@ -3,18 +3,19 @@
 /**
  * Possible networks to resolve.
  */
+@SuppressWarnings("checkstyle:AbbreviationAsWordInName")
```

Comment:
What warning is this? I see this in a lot of files, can you just disable whatever rule this is?

Reply:
Oh sorry, my PR description should have mentioned this. I filed https://app.asana.com/0/1143722888069470/1170259834689737 to track. 

This is yelling at me to not abbreviate things in names. It wants the classes to be named `XrpClient` instead of `XRPClient` or `PayId` rather than `PayID`. 

I still haven't decided on a standard yet, and changing class names (and or changing iVar names inside of classes inconsistently) feels bad so I'm basically just deferring the problem, for now. 

I'd like this to not get worse though, so I've left the rule enabled globally.